packages/quantic/force-app/main/default/lwc/quanticFacetValue/quanticFacetValue.html

```diff
@@ -24,6 +24,7 @@
     <template if:true={isChecked}>
       <span
         class="slds-truncate slds-var-m-right_xx-small slds-var-m-top_xxx-small nowrap facet__value-text facet__value_selected"
+        title={formattedFacetValue}
         >{formattedFacetValue}</span
       >
       <span class="nowrap slds-var-m-top_xxx-small facet__number-of-results facet__value_selected"
```

Comment:
can we remove this `nowrap` CSS class? I can't find it defined anywhere.
```suggestion
      <span class="slds-var-m-top_xxx-small facet__number-of-results facet__value_selected"
```

Reply:
yeah i think the correct one would be `slds-nowrap` but clearly we don't need it either. Will do.